connectors/rsk.go

```diff
@@ -293,6 +294,10 @@ func (rsk *RSK) RegisterPegIn(opt *bind.TransactOpts, q bindings.LiquidityBridge
 	return rsk.lbc.RegisterPegIn(opt, q, signature, btcRawTrx, partialMerkleTree, height)
 }
 
+func (rsk *RSK) RegisterPegInWithoutTx(q bindings.LiquidityBridgeContractQuote, signature []byte, tx []byte, pmt []byte, height *big.Int, txHash string) error {
+	return rsk.lbc.RegisterPegInWithoutTx(q, signature, tx, pmt, height, txHash)
+}
+
```

Comment:
This could be the same function as RegisterPegIn with an argument to decide whether to mine the transaction (using LBCCaller or LBC depending on the case).

Reply:
I'd rather leaving it as it is for 2 main reasons:
1) making both signatures to match would imply 2 option params to be passed in or passing an useless transactopts to the function that uses the caller (and builds a callopts on the fly).
2) you can deduce what the function does based on its name, and looking it from the code that uses them it's easier to see what's intended.